### DISCUSSION TITLE
Galaxy badge question

### DISCUSSION THREAD

**Discussion by rbento01** [community/community#33719] [Code Security]
Do you have to get the correct answer ?

**Comment by JMatos38** [comment]
Falta informação para fornecer uma resposta adequada. Pode descrever melhor

**Comment by Murat-Karakaya** [comment]
To earn the Galaxy Brain badge, you need to provide answers to discussions. If you choose a comment from this discussion as the answer, the person who wrote that comment will either make progress toward getting the badge or will outright earn it.

To get the badge, you should have at least two comments that are chosen as the accepted answer.

If this is not what you meant, then please let me know :)

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by raghvendragoayal** [comment]
- [x] https://github.com/cameronmcnz/rock-paper-scissors.git

**Comment by raghvendragoayal** [comment]
Here’s what the three GitHub URLs look like if you were to copy each of them:
